src/tg/index.js

```diff
@@ -64,11 +71,11 @@ var init = function(msgCallback) {
             }
 
             if (message.user) {
-                message.text = '<' + message.user + '> ' + message.text;
+                message.text = '<' + message.user + '> ' + irc_to_html_format(message.text);
```

Comment:
Surely if you're sending HTML tags in a message you'd want to escape where it shows the user who talked? It might be mistaken as a tag.